outputs.tf

```diff
@@ -1,24 +1,29 @@
 output "instances" {
   description = "List of instance IDs"
-  value       = ["${aws_instance.instance.*.id}"]
+  value       = ["${module.dcos-windows-instances.instances}"]
 }
 
 output "public_ips" {
   description = "List of public ip addresses created by this module"
-  value       = ["${aws_instance.instance.*.public_ip}"]
+  value       = ["${module.dcos-windows-instances.public_ips}"]
 }
 
 output "private_ips" {
   description = "List of private ip addresses created by this module"
-  value       = ["${aws_instance.instance.*.private_ip}"]
+  value       = ["${module.dcos-windows-instances.private_ips}"]
 }
 
 output "os_user" {
   description = "The OS user to be used"
-  value       = "${module.dcos-tested-oses.user}"
+  value       = "Administrator"
 }
 
 output "prereq-id" {
```

Comment:
can be removed

Reply:
just a clone of other modules but true...